complete-guide-to-writing-winning-demo-scripts.mdx

```diff
@@ -6,26 +6,12 @@ bannerImg: https://github.com/sid-patri-fable/fable-blog/assets/155963795/17c6cf
 ogTitle: A Complete Guide to Writing Winning Demo Scripts
 ogDescription: Writing a compelling demo script is key to winning over your audience. This complete guide provides you with all the tips and tricks you need to create a winning demo script that will impress your prospects.
 ogImage: https://github.com/sid-patri-fable/fable-blog/assets/155963795/17c6cff0-0722-4680-9baa-ddc9e7597701
+bannerDate: January 12, 2024
+bannerTitle: A Complete Guide to Writing Winning Demo Scripts
+bannerSubtitle: A step-by-step guide for planning, creating, and preparing a successful demo script for sales teams.
 ---
 
-import Banner from './.components/banner'
-import Promotion from './.components/Promotion'
 import Content from './.components/Content'
-import CoverImg from './.components/CoverImg'
-import PromotionBannerCta from './.components/PromotionBannerCta'
-
-export const bannerDate = "January 12, 2024"
-export const bannerTitle = "A Complete Guide to Writing Winning Demo Scripts" 
-export const bannerSubtitle = "A step-by-step guide for planning, creating, and preparing a successful demo script for sales teams."
-
-<Banner config={props.config}>
-  <div style={{ maxWidth: '768px' }}>
-    {bannerDate}
-    # {bannerTitle}
-    {bannerSubtitle}
-  </div>
-  <CoverImg src="https://github.com/sid-patri-fable/fable-blog/assets/155963795/17c6cff0-0722-4680-9baa-ddc9e7597701" />
-</Banner>
 
 <Content toc={props.toc}>
```

Comment:
this `<Content />` should also be a part of customLayout
- u can pass toc as props the same way u did for frontmatter